File: specification/vmware/resource-manager/Microsoft.AVS/AVS/stable/2025-09-01/examples-java/PrivateClouds_Update_Stretched.java

import com.azure.resourcemanager.avs.models.ManagementCluster;
import com.azure.resourcemanager.avs.models.PrivateCloud;

/**
 * Samples for PrivateClouds Update.
 */
public final class Main {
    /*
     * x-ms-original-file: 2025-09-01/PrivateClouds_Update_Stretched.json
     */
    /**
     * Sample code: PrivateClouds_Update_Stretched.
     * 
     * @param manager Entry point to AvsManager.
     */
    public static void privateCloudsUpdateStretched(com.azure.resourcemanager.avs.AvsManager manager) {
        PrivateCloud resource = manager.privateClouds()
            .getByResourceGroupWithResponse("group1", "cloud1", com.azure.core.util.Context.NONE).getValue();
        resource.update().withManagementCluster(new ManagementCluster().withClusterSize(4)).apply();
    }
}

File: specification/vmware/resource-manager/Microsoft.AVS/AVS/stable/2025-09-01/examples-js/PrivateClouds_Update_Stretched.js
const { AzureVMwareSolutionAPI } = require("@azure/arm-avs");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to update a PrivateCloud
 *
 * @summary update a PrivateCloud
 * x-ms-original-file: 2025-09-01/PrivateClouds_Update_Stretched.json
 */
async function privateCloudsUpdateStretched() {
  const credential = new DefaultAzureCredential();
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const client = new AzureVMwareSolutionAPI(credential, subscriptionId);
  const result = await client.privateClouds.update("group1", "cloud1", {
    properties: { managementCluster: { clusterSize: 4 } },
  });
  console.log(result);
}
